proposals/json-td/index.html

```diff
@@ -112,16 +142,33 @@
         <p>The events member is a map of event objects which define the types of events which may be emitted by a device when a certain condition is met. Examples might include a temperature sensor or fill level sensor reaching a certain threshold.</p>
       </section>
       <section>
+        <h3><code>links</code> member</h3>
+        <p>The links member provides a map of URLs for Properties, Actions and Events resources. It can also contain a WebSocket URL to be used for the Thing's WebSocket API. Entries in this map have a key which can be one of <code>properties</code>, <code>actions</code>, <code>events</code> and a string value representing a URL.</p>
+      </section>
+      <section>
         <h3><code>Property</code> object</h3>
-        <p>A property object describes an attribute of a Thing and is indexed by a property name. A property description may include its type, unit, minimum value, maximimum value, writability, description and a reference to a URL for getting and setting the value of the property.</p>
+        <p>A property object describes an attribute of a Thing. It contains a <code>href</code> reference to a URL for getting the value of the property as well as all the fields of a <a href="#value-object"><code>value</code> object</a>.</p>
       </section>
       <section>
         <h3><code>Action</code> object</h3>
-        <p>An action object describes a function which can be carried out on a device. An action description may include action parameters and a human friendly description which describes what the action does.</p>
+        <p>An action object describes a function which can be carried out on a device. An action may include parameters as a <code>data</code> field containing a <a href="#value-object">value object</a>.
       </section>
       <section>
         <h3><code>Event</code> object</h3>
-        <p>An event object describes a type of event which may be emitted by a device. This may include the type of values which may be included in the event and a human friendly description which describes what the event means.</p>
+        <p>An event object describes a type of event which may be emitted by a device. An event includes parameters as a <code>data</code> field containing a <a href="#value-object">value object</a>. An event may also have <code>name</code> and a human friendly <code>description</code> which describes what the event was about.
```

Comment:
Again, "human readable" rather than "human friendly" is probably less ambiguous.